org.adempiere.install/src/org/compiere/install/Setup.java

```diff
@@ -151,17 +149,18 @@ else if (e.getSource() == menuHelpInfo)
 	 */
 	public static void main(String[] args)
 	{
-		CLogMgt.initialize(true);
-		Handler fileHandler = new CLogFile(System.getProperty("user.dir"), false, false);
-		CLogMgt.addHandler(fileHandler);
+		CLogMgt.initialize(false);
+		// Duplicate log file - not required
+		// Handler fileHandler = new CLogFile(System.getProperty("user.dir"), false, false);
+		// CLogMgt.addHandler(fileHandler);
 		//	Log Level
 		if (args.length > 0)
 			CLogMgt.setLevel(args[0]);
 		else
 			CLogMgt.setLevel(Level.INFO);
 		//	File Logger at least FINE
-		if (fileHandler.getLevel().intValue() > Level.FINE.intValue())
-			fileHandler.setLevel(Level.FINE);
+		// if (fileHandler.getLevel().intValue() > Level.FINE.intValue())
+		//	fileHandler.setLevel(Level.FINE);

```

Comment:
with history from git, it seems strange to me that we will still do remark instead of remove.